rollup.config.js

```diff
@@ -28,11 +34,15 @@ const useLodashEs = () => ({
   }
 });
 
-const common = {
-  input: "src/index.js"
-};
+const inputAsDir = () => ({
+  name: "inputAsDir",
+  resolveId: path => {
+    const dir = join(path, "index.js");
+    return existsSync(dir) ? dir : null;
```

Comment:
We need this rollup plugin in order to make `input` accept directories. It'll be removed when https://github.com/rollup/rollup-plugin-commonjs/pull/329 lands.

Reply:
u dont need to run `commonjs` over ur entry files at all, just do this
```js
		commonjs({
			include: [
				'node_modules/**',
			],
		}),
```
U shouldnt find any non-ESM files inside ur own project

Reply:
and u can use this plugin only for UMD bundles